docs/modules/documentation/containers/alert/alert-docs.component.html

```diff
@@ -1,56 +1,72 @@
 <header>Alert</header>
-<description>Alerts are used to display short messages that require the user's attention.</description>
+<description>Alerts are used to display short messages that require user attention.</description>
 <import module="AlertModule" path="fundamental-ngx"></import>
 <separator></separator>
 
 <h1>&lt;fd-alert&gt;</h1>
 
 <properties [properties]="{
   inputs: [
-    { name: 'type', description: 'Determines the type of alert - \'error\' (red) or \'warning\' (orange). Defaults to white if no type is provided.' },
-    { name: 'id', description: 'The id of the alert. If omitted, an id will be generated by a HashService.' },
-    { name: 'dismissible', description: 'Shows a dismissible button if set to true. Default is false.' },
-    { name: 'persist', description: 'Boolean, when set to true, the alert will not disappear after a set amount of time.  Default is false.'},
-    { name: 'mousePersist', description: 'Boolean, when set to true, the alert will remain visible as long as the mouse is hovered over it.  The alert will disappear when the mouse leaves the alert.  Default is false.'},
-    { name: 'visibleTime', description: 'Number dictating how long (in milliseconds) the alert will be visible.  Default is 10000.'}
+    { name: 'type', description: 'String. Determines the type of alert. View example below for options. Defaults to white if no type is provided.' },
+    { name: 'id', description: 'String. The id of the alert. If omitted, an id will be generated.' },
+    { name: 'dismissible', description: 'Boolean. Shows a dismiss button. Default is true.' },
+    { name: 'persist', description: 'Boolean. When set to true, the alert will not disappear after a set amount of time.  Default is false.'},
+    { name: 'mousePersist', description: 'Boolean. When set to true, the alert will remain visible as long as the mouse is hovered over it. Default is false.'},
+    { name: 'duration', description: 'Number. Dictates how long in milliseconds the alert will be visible. Default is 10000.'},
+    { name: 'ariaLabel', description: 'String. Aria-label attribute for the alert component.' },
+    { name: 'ariaLabelledBy', description: 'String. Aria-labelledby attribute for the alert component.' },
+    { name: 'width', description: 'String. Width of the alert. Supports all css units.' },
```

Comment:
maybe we can show some examples

Reply:
Not sure if that's super useful, it just changes the width?